addons/website_event_questions/controllers/main.py

```diff
@@ -9,10 +9,13 @@ class WebsiteEvent(WebsiteEventController):
     def _process_attendees_form(self, event, form_details):
         ''' Process data posted from the attendee details form. '''
         registrations = super(WebsiteEvent, self)._process_attendees_form(event, form_details)
+
+        answer_ids = []
+        for key, value in form_details.items():
```

Comment:
I have a doubt ... normally questions can be either individual, either for everyone. It seems here answers will go to every registrations which is probably not what we want. Did you check ?

Reply:
@tde-banana-odoo 
Hum yes you are right, you can choose if the question is general or not, I fix that